lumigator/backend/backend/services/jobs.py

```diff
@@ -430,7 +451,7 @@ def create_job(
         settings.inherit_ray_env(runtime_env_vars)
 
         # set num_gpus per worker (zero if we are just hitting a service)
-        if not request.model.startswith("hf://"):
+        if job_type == JobType.INFERENCE and not request.job_config.model.startswith("hf://"):
             worker_gpus = job_settings["ray_worker_gpus_fraction"]
         else:
             worker_gpus = job_settings["ray_worker_gpus"]
```

Comment:
I don't think it needs to be done in this PR necessarily, but I wonder if this logic can be moved to generate_inference_job_config

Reply:
This would be part of the general separation between job specific config and job-backend interaction config, so I'd leave it for another PR.